yt/loaders.py

```diff
@@ -1015,7 +1019,11 @@ def load_octree(
     # for compatibility
     if over_refine_factor is not None:
         nz = 1 << over_refine_factor
-    domain_dimensions = np.array([nz, nz, nz])
+    if domain_dimensions is None:
+        # We assume that if it isn't specified, it defaults to the number of
+        # zones (i.e., a single root oct.)
+        domain_dimensions = [nz, nz, nz]
+    domain_dimensions = np.array(domain_dimensions)
```

Comment:
To reduce the type variability of `domain_dimensions`
```suggestion
    if domain_dimensions is None:
        # We assume that if it isn't specified, it defaults to the number of
        # zones (i.e., a single root oct.)
        domain_dimensions = np.array([nz, nz, nz])
    else:
        domain_dimensions = np.array(domain_dimensions)
```

Reply:
I don't like this change, to be honest.  Both seem clear, and I think my version is fine, as the pass-through is *right* there -- the distance is the same.

Reply:
That's pretty irritating.